### PR TITLE
niclake Boardwalk layout

### PR DESCRIPTION
**Pull request by niclake**
## Description

Creation of my own layout/keymap for the Boardwalk keyboard

## Types of Changes

- [X] Keymap/layout/userspace (addition or update)

### CODE REVIEW

File: keyboards/boardwalk/keymaps/niclake/config.h [hunk 1]
```diff
@@ -0,0 +1 @@
+#define TAPPING_TOGGLE 2
```
Comment (fauxpark) [inline keyboards/boardwalk/keymaps/niclake/config.h:1]:
```suggestion
#pragma once

#define TAPPING_TOGGLE 2
```

File: keyboards/boardwalk/keymaps/niclake/rules.mk [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
```
Comment (fauxpark) [inline keyboards/boardwalk/keymaps/niclake/rules.mk:13]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/boardwalk/keymaps/niclake/config.h [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#pragma once
```
Comment (noroadsleft) [inline keyboards/boardwalk/keymaps/niclake/config.h:1]: This file should have a license header here, as `keymap.c` does.

**Review by noroadsleft — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**